### ISSUE TITLE
activateNewNote() problems

### ISSUE THREAD

**Issue by wwwald** [zadam/trilium#1153]
Running Trilium 0.43.2. 

I have scripts for creating children notes for the day note, that are assigned a certain pre-defined template.
Here's an example:

```javascript
api.addButtonToToolbar({
    title: 'Meeting note',
    icon: 'check',
    shortcut: 'alt+m',
    action: async () => {
        const currentNote = api.getActiveTabNote();
        const dtstamp = getDateTimeISO(new Date()); // this returns a YYYY-MM-DD HH:MM formatted timestamp
        const meetingNote = await api.runOnServer(async (parentId, dt) => {
            const {note: meetingNote} = await api.createTextNote(parentId, dt + " ", "");
            const meetingTemplate = await api.searchForNote('@title=template-meeting');
            await meetingNote.addAttribute("relation", "template", meetingTemplate.noteId, false);
            return meetingNote;
        }, [currentNote.noteId, dtstamp]);
        await api.waitUntilSynced();
        await api.activateNewNote(meetingNote.noteId);
    }
});
```

This script used to work as expected (or as _I_ expected) before, but now doesn't anymore. The note gets created and assigned the template relation, but afterwards does not get selected in the treeview.

Any idea on a fix would be much appreciated!

**Comment by zadam** [comment]
Hello, I tried your script and it works well - it does activate the note after creating it.

Just for info, you say that it worked before - on which version did it work the last time?

Also could you post log from the console (CTRL-SHIFT-I) there could be some hint that something went wrong...

**Comment by wwwald** [comment]
That must have been in 0.42.7, not sure where in 0.43.0beta / 0.43.1 it actually stopped.

I also notice that after triggering the script, Trilium systematically refreshes due to a timeout in `desktop.js`.

The log (with some <<< ----------  indications ---------- >>> of key steps added):
```
<<< ---------- Trilium refresh ---------- >>>**
DevTools failed to load SourceMap: Could not load content for http://127.0.0.1:37840/libraries/bootstrap/js/bootstrap.bundle.min.js.map: Unknown error
DevTools failed to load SourceMap: Could not load content for http://127.0.0.1:37840/libraries/fancytree/jquery.fancytree-all-deps.min.js.map: Unknown error
DevTools failed to load SourceMap: Could not load content for http://127.0.0.1:37840/libraries/split.min.js.map: Unknown error
DevTools failed to load SourceMap: Could not load content for http://127.0.0.1:37840/libraries/bootstrap/css/bootstrap.min.css.map: Unknown error
VM960 desktop.js:1 Cannot find action logout
VM960 desktop.js:1 Cannot find action createTopLevelNote
VM960 desktop.js:1 Cannot find action renderActiveNote
<<< ---------- Trigger script via shortcut ---------- >>>
VM960 desktop.js:1 Uncaught (in promise) TypeError: Cannot read property 'noteId' of undefined
    at o.isAffecting (VM960 desktop.js:1)
    at VM960 desktop.js:1
    at Array.find (<anonymous>)
    at J.entitiesReloadedEvent (VM960 desktop.js:1)
    at J.callMethod (VM960 desktop.js:1)
    at VM960 desktop.js:1
    at async Promise.all (:37840/index 0)
    at async Promise.all (:37840/index 0)
    at async Promise.all (:37840/index 1)
    at async Promise.all (:37840/index 1)
isAffecting @ VM960 desktop.js:1
(anonymous) @ VM960 desktop.js:1
entitiesReloadedEvent @ VM960 desktop.js:1
callMethod @ VM960 desktop.js:1
(anonymous) @ VM960 desktop.js:1
Promise.then (async)
(anonymous) @ VM960 desktop.js:1
timeLimit @ VM960 desktop.js:1
(anonymous) @ VM960 desktop.js:1
_ @ VM960 desktop.js:1
VM960 desktop.js:1 09:47:14 Encountered error Process exceeded time limit 5000: Error: Process exceeded time limit 5000
    at http://127.0.0.1:37840/app-dist/desktop.js:1:70362, reloading frontend.
VM960 desktop.js:1 console.trace
y @ VM960 desktop.js:1
(anonymous) @ VM960 desktop.js:1
async function (async)
(anonymous) @ VM960 desktop.js:1
_ @ VM960 desktop.js:1
VM966:25 -----------
VM966:26 {noteId: "Oxu9sZbBb1Gg", title: "2020-07-17 09:47 ", isProtected: false, type: "text", mime: "text/html", …}
VM960 desktop.js:1 09:47:14 Did not find parent root for child Oxu9sZbBb1Gg
VM960 desktop.js:1 09:47:14 Available parents: [d]
DevTools failed to load SourceMap: Could not load content for http://127.0.0.1:37840/libraries/ckeditor/ckeditor.js.map: Unknown error
<<< ---------- Trilium refresh got triggered ---------- >>>
Navigated to http://127.0.0.1:37840/
DevTools failed to load SourceMap: Could not load content for http://127.0.0.1:37840/libraries/bootstrap/js/bootstrap.bundle.min.js.map: Unknown error
DevTools failed to load SourceMap: Could not load content for http://127.0.0.1:37840/libraries/fancytree/jquery.fancytree-all-deps.min.js.map: Unknown error
DevTools failed to load SourceMap: Could not load content for http://127.0.0.1:37840/libraries/split.min.js.map: Unknown error
DevTools failed to load SourceMap: Could not load content for http://127.0.0.1:37840/libraries/bootstrap/css/bootstrap.min.css.map: Unknown error
keyboard_actions.js:76 Cannot find action logout
keyboard_actions.js:76 Cannot find action createTopLevelNote
keyboard_actions.js:76 Cannot find action renderActiveNote

```

**Comment by zadam** [comment]
Hi, thanks for the logs, this looks exactly like issue described in the last comments of #803 which was fixed in stable branch in 3d324b9 so it's going to be in the next stable release.

Thanks for help!
